README.md

```diff
@@ -134,8 +134,8 @@ This section explains the basic usage of Windows Remote Arduino. This is an exce
         using namespace Microsoft.Maker.Serial;
 
 		//create a bluetooth connection and pass it to the RemoteDevice
-		IStream bt = new BluetoothSerial();
-        RemoteDevice arduino = new RemoteDevice( bt );
+		IStream connection = new BluetoothSerial( "MyBluetoothDevice" );
```

Comment:
I don't see where in the documentation "MyBluetoothDevice" is described. Why is this required? Also, I noticed that it is not consistent in the documentation. The Advanced readme uses the previous syntax